core/src/main/scala/org/elasticmq/MessageSystemAttribute.scala

```diff
@@ -0,0 +1,14 @@
+package org.elasticmq
+
+import java.util.Base64
+
+sealed trait MessageSystemAttribute
```

Comment:
I think this can be defined in the `rest` module, as that's the only place where it's used?

Reply:
Indeed, moved to `rest` module